sklearn/tests/test_calibration.py

```diff
@@ -341,3 +349,53 @@ def decision_function(self, X):
     calibrated_clf = CalibratedClassifierCV(MockTensorClassifier())
     # we should be able to fit this classifier with no error
     calibrated_clf.fit(X, y)
+
+
+# Create prefit pipeline
```

Comment:
If you want to use this several time, it will be cleaner to use a `pytest.fixture`